ui/hooks/useTokensToSearch.js

```diff
@@ -66,7 +66,8 @@ export function getRenderableTokenData(
   const usedIconUrl =
     iconUrl ||
     (tokenList[tokenAddress] &&
-      `images/contract/${tokenList[tokenAddress].iconUrl}`);
+      `images/contract/${tokenList[tokenAddress].iconUrl}`) ||
+    token?.image;
```

Comment:
The `token` is destructured earlier in this function, so `token` can't be falsy here.
```suggestion
    token.image;
```